src/app/pages/storage/modules/pool-manager/components/pool-manager-wizard/components/layout-step/automated-disk-selection/automated-disk-selection.component.spec.ts

```diff
@@ -327,4 +330,30 @@ describe('AutomatedDiskSelection', () => {
       expect(await widthSelect.getOptionLabels()).toStrictEqual(['1', '2', '3', '4', '5', '6', '7', '8', '9']);
     });
   });
+
+  it('resets form if Start Over confirmed', async () => {
+    await layoutSelect.setValue('Stripe');
+    await sizeSelect.setValue('12 TiB (HDD)');
+
+    const form = spectator.component.form;
+
+    form.patchValue({ treatDiskSizeAsMinimum: true });
+
+    expect(form.value).toStrictEqual({
+      layout: CreateVdevLayout.Stripe,
+      sizeAndType: [13194139533312, DiskType.Hdd],
+      treatDiskSizeAsMinimum: true,
+      width: null,
+    });
+
+    const store = spectator.inject(PoolManagerStore);
+    store.startOver$.next();
```

Comment:
Instead of testing like this, we're supposed to test the front-end inputs. Click the button and then check if the desired action is performed with testScheduler etc.